package.json

```diff
@@ -31,14 +31,11 @@
     "version": "npx conventional-changelog-cli -p angular -i CHANGELOG.md -s && git add CHANGELOG.md",
     "postversion": "git push && git push --tags"
   },
-  "dependencies": {
-    "bootstrap": "^3.3.7",
```

Comment:
They dependencies were defined in order to use them inside a .less file I guess.

ref: https://github.com/ovh-ux/ng-ovh-web-universe-components/blob/master/src/v6UiSwitch/v6-ui-switch.less#L2

Reply:
You guess correctly :p

I saw that, less are builded with the dist, and dist is builded in dev mode. If you check what I have done, I move only dependencies from dependencies to devDependencies, to avoid installing them when we add theses dependencies in the host project.

So, it will break nothing in UI, we don't deliver less in production.